Userland/Libraries/LibJS/Runtime/OrdinaryFunctionObject.cpp

```diff
@@ -246,20 +234,13 @@ Value OrdinaryFunctionObject::construct(FunctionObject&)
     return execute_function_body();
 }
 
-JS_DEFINE_NATIVE_GETTER(OrdinaryFunctionObject::length_getter)
-{
-    auto* function = typed_this(vm, global_object);
-    if (!function)
-        return {};
-    return Value(static_cast<i32>(function->m_function_length));
-}
-
-JS_DEFINE_NATIVE_GETTER(OrdinaryFunctionObject::name_getter)
+void OrdinaryFunctionObject::set_name(const FlyString& name)
```

Comment:
Tempted to suggest we go even further and make that https://tc39.es/ecma262/#sec-setfunctionname, but up to you.

Reply:
right, i thought about doing that, but this (and most of the rest of function constructor) is not very conforming, so i have a feelign this function would be replaced by something else when we decide to make it conforming anyways